ssg-rhel8-ds-justifications.yaml

```diff
@@ -0,0 +1,279 @@
+
+  - rule_id: xccdf_org.ssgproject.content_rule_mount_option_tmp_nosuid
+    severity: medium
+    - justification: |
+        Not partitioned, this does not apply to short-lived CloudFormation instances and will be tailored.
+
+  - rule_id: xccdf_org.ssgproject.content_rule_mount_option_var_log_audit_nodev
+    severity: medium
+    - justification: |
+        Not partitioned, this does not apply to short-lived CloudFormation instances and will be tailored.
+
+  - rule_id: xccdf_org.ssgproject.content_rule_mount_option_var_log_audit_noexec
+    severity: medium
+    - justification: |
+        Not partitioned, this does not apply to short-lived CloudFormation instances and will be tailored.
+
+  - rule_id: xccdf_org.ssgproject.content_rule_mount_option_var_log_audit_nosuid
+    severity: medium
+    - justification: |
+        Not partitioned, this does not apply to short-lived CloudFormation instances and will be tailored.
+
+  - rule_id: xccdf_org.ssgproject.content_rule_mount_option_var_log_nodev
+    severity: medium
+    - justification: |
+        Not partitioned, this does not apply to short-lived CloudFormation instances and will be tailored.
+
+  - rule_id: xccdf_org.ssgproject.content_rule_mount_option_var_log_noexec
+    severity: medium
+    - justification: |
+        Not partitioned, this does not apply to short-lived CloudFormation instances and will be tailored.
+
+  - rule_id: xccdf_org.ssgproject.content_rule_mount_option_var_log_nosuid
+    severity: medium
+    - justification: |
+        Not partitioned, this does not apply to short-lived CloudFormation instances and will be tailored.
+
+  - rule_id: xccdf_org.ssgproject.content_rule_mount_option_var_tmp_nodev
+    severity: medium
+    - justification: |
+        Not partitioned, this does not apply to short-lived CloudFormation instances and will be tailored.
+
+  - rule_id: xccdf_org.ssgproject.content_rule_mount_option_var_tmp_noexec
+    severity: medium
+    - justification: |
+        Not partitioned, this does not apply to short-lived CloudFormation instances and will be tailored.
+
+  - rule_id: xccdf_org.ssgproject.content_rule_mount_option_var_tmp_nosuid
+    severity: medium
+    - justification: |
+        Not partitioned, this does not apply to short-lived CloudFormation instances and will be tailored.
+
+  - rule_id: xccdf_org.ssgproject.content_rule_sssd_enable_certmap
+    severity: medium
+    - justification: |
+        Image intended for FedRAMP cloud usage, no federated identity used, and will be tailored.
+
+  - rule_id: xccdf_org.ssgproject.content_rule_usbguard_generate_policy
+    severity: medium
+    - justification: |
+        CloudFormation instances do not have USB devices and will be tailored.
+
+  - rule_id: xccdf_org.ssgproject.content_value_var_accounts_authorized_local_users_regex
+    severity: medium
+    - justification: |
+        Authorized user list customized based on GlobalNET base configuration.
```

Comment:
Sorry, didn't catch this before - we should remove GlobalNET from the Hailstone justification. It's just the default local user list that we're customizing for.